tests/neo.UnitTests/Trie/MPT/UT_MPTTrie.cs

```diff
@@ -0,0 +1,373 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.IO;
+using Neo.Persistence;
+using Neo.Trie.MPT;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Neo.UnitTests.Trie.MPT
+{
+    public class TestKey : ISerializable
+    {
+        private byte[] key;
+
+        public int Size => key.Length;
+
+        public TestKey()
+        {
+
```

Comment:
```suggestion
            this.key = Array.Empty<byte>();
```

You have a public constructor and this could set the key as null, but you always use the key without checking if it's null.